share/scan/registry/manifest.go

```diff
@@ -46,6 +46,11 @@ func (r *Registry) ManifestRequest(ctx context.Context, repository, reference st
 	retry := 0
 	withOCIManifest := false
 	withOCIIndex := false
+
```

Comment:
Please add a parameter to ManifestRequest(), like ManifestRequest(....., forSigImage bool) so that forSigImage also needs to be true for line 51 to be executer

Reply:
Implemented as `ManifestRequestType`.